config.go

```diff
@@ -698,6 +698,18 @@ func loadConfig() (*config, error) {
 		defaultChainSubDirname,
 		decredChain.String())
 
+	// Update Dcrwallet.GRPCHost with correct port from
+	// activeNetParam selected
+	if cfg.Dcrwallet.GRPCHost != "" && !strings.Contains(
+		cfg.Dcrwallet.GRPCHost,
+		":") {
```

Comment:
This breaks for IPv6 addresses. This needs to be based on `net.SplitHostPort` so that it is consistent with other decred software.

For example, this is the start of the call stack for filling the port in `dcrctl`: https://github.com/decred/dcrd/blob/master/cmd/dcrctl/config.go#L348